daemon/main.go

```diff
@@ -240,6 +240,8 @@ func init() {
 		"agent-labels", []string{}, "Additional labels to identify this agent")
 	flags.StringVar(&config.AllowLocalhost,
 		"allow-localhost", AllowLocalhostAuto, "Policy when to allow local stack to reach local endpoints { auto | always | policy } ")
+	flags.Bool(
+		"auto-v6-l3-routes", false, "Automatically adds IPv6 L3 routes to reach other nodes for non-overlay mode (--device) (BETA)")
```

Comment:
Let's rename this to `--auto-ipv6-node-routes`

Reply:
Done!